src/server/scripts/Northrend/VioletHold/instance_violet_hold.cpp

```diff
@@ -199,16 +201,21 @@ class instance_violet_hold : public InstanceMapScript
                     break;
                 default:
                     break;
+                case NPC_VOID_SENTRY:
```

Comment:
Sentries and balls are despawned when he dies at https://github.com/TrinityCore/TrinityCore/pull/14860/files#diff-8c3ab39be0479fd288b3fb18cf8fe49dR155

In which scenarios would they still be created after his death?